bebop_driver/data/bebop2_camera_calib.yaml

```diff
@@ -0,0 +1,20 @@
+image_width: 640
```

Comment:
Could you please add an optional argument to `bebop_driver` launch files so someone can switch between *Bebop* 1 and 2. Based on that argument those launch files can load the appropriate camera calibration params.